hedera-node/docs/system-accounts-operations.md

```diff
@@ -0,0 +1,184 @@
+# System accounts
+
+## Definition
+The Hedera network reserves the first 1000 entity numbers for its own uses, and any account with a number in this reserved range is referred to as a system account.
+
+The following addresses are reserved for the Hedera precompiled contracts:
+- 0x167 (0.0.359) - HTS precompile;
+- 0x168 (0.0.360) - ExchangeRate precompile;
+- 0x169 (0.0.361) - PRNG precompile.
+
+Some of the system accounts exist today and more can be created in the future. Certain system accounts have predefined roles in the network. These include:
+- 0.0.2 - treasury account;
+- 0.0.50 - system admin;
+- 0.0.55 - address book admin;
+- 0.0.57 - exchange rates admin;
+- 0.0.58 - freeze admin;
+- 0.0.59 - system delete admin;
+- 0.0.60 - system undelete admin;
+- 0.0.800 - staking reward account;
+- 0.0.801 - node reward account.
+
+Addresses in range 1 → 9 are special addresses on which the Ethereum precompiled contracts exist. This is valid for both Ethereum and Hedera.
+
+Lastly, the system accounts can be divided in these two groups:
+- addresses in range 0 → 750 - accounts that reject hbar transfers. An exception is the HTS precompiled contract that can receive transfers.
+- addresses in range 751 → 1000 - accounts which may receive transfers.
+
+## Rationale
+As a result of the specifics of the system accounts in Hedera described in the section above there are some differences when calling operations on system accounts in Hedera and in Ethereum via smart contract executions.
+This doc covers these differences and describes the expected behavior.
+
+## Operations
+
+Legend of the symbols used in the tables below:
+- ✅ - the operation was successful;
+- ❌ - the operation failed;
+- ❗ - something specific to note when executing an operation.
+
+### ExtCode operations
+
+`ExtCodeSize`
+
+**Ethereum:** Get size of an account’s code. If there’s no code, it returns 0.
+
+**Hedera:** For addresses in range 0x0 → 0x3E8 (0.0.0 → 0.0.1000) it returns 0.
+
+`ExtCodeCopy`
+
+**Ethereum:** Copy an account’s code to memory. If a contract doesn’t exist on the address, it copies empty bytes.
+
+**Hedera:** For addresses in range 0x0 → 0x3E8 (0.0.0 → 0.0.1000) it copies empty bytes.
+
+`ExtCodeHash`
+
+**Ethereum:** Hash of the chosen account's code, the empty hash (0xc5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470) if the account has no code, or 0 if the account does not exist or has been destroyed.
+
+**Hedera:** For addresses in range 0x0 → 0x3E8 (0.0.0 → 0.0.1000) it returns the empty hash.
```

Comment:
empty or 0x hash?

Reply:
The long-zero address actually. The current text is misleading in this case as it looks like the empty hash is the same as in Ethereum but it isn't. I will update this.